workers/ohsome_quality_analyst/utils/helper.py

```diff
@@ -181,3 +182,9 @@ def load_sklearn_model(path: str):
     model = joblib.load(path)
     warnings.resetwarnings()
     return model
+
+
+async def sem_task(task: Coroutine, semaphore=asyncio.Semaphore(4)) -> Coroutine:
```

Comment:
I would prefer to have the default definable by configuration. But 4 is a decent default :+1:

Reply:
Added this as TODO in #255